config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql

```diff
@@ -0,0 +1,33 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX bestuursfunctieCode: <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/>
+
+DELETE {
+  GRAPH ?graph {
+  ?mandataris mandaat:rangorde ?rangorde.
+  }
+}
+INSERT {
+  GRAPH ?graph {
+    ?mandataris mandaat:rangorde ?newLabel.
+  }
+}
+WHERE {
```

Comment:
this query didn't work for me when i ran it on a prod backup

Reply:
as in: there were cases of Toegevoegde schepen still left in the data

Reply:
verify with:
```
PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
SELECT * WHERE {
  GRAPH ?g {
  ?s mandaat:rangorde ?rank.
  FILTER(CONTAINS(?rank, "Toegevoegde schepen"))
  }
} 
```

Reply:
The result of your query is now with the update
```
[
	{
		"rank": "Toegevoegde schepen "
	},
	{
		"rank": "Toegevoegde schepen (derde)"
	}
]
```

The first one should be fixed by the rangorde input and the second one is something a user should fix IMO

Reply:
Both queries now return no results so every match is handled, if the latest to load data is somewhat the same as on PROD it should handle it. If not i will just have to add ALL bestuursfunctie codes to the Values statement..

Reply:
Will create a ticket for this